.config/gitpod/Dockerfile

```diff
@@ -36,6 +40,12 @@ RUN PYTHON_VERSION=$(cat /tmp/python_version.config | cut -d '=' -f 2) && \
     $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null \
     && apt-get update \
     && apt-get install -y --no-install-recommends docker-ce docker-ce-cli containerd.io \
+    ## Install npm
```

Comment:
@ThomasSanson could you describe why we need to install npm?

Reply:
Hi @vitabaks,

Thank you for your review.

To answer your first question about the need for npm, the installation of Node.js and npm was suggested to improve the Developer Experience (DX) for our project that utilizes Gitpod. Currently, developers have to manually install Node.js every time they start a new workspace, which can be time-consuming and frustrating. By including Node.js and npm in our Dockerfile, we ensure that these tools are automatically installed and available as soon as developers start their Gitpod workspace. This not only saves time but also promotes consistency across the development environment by having all developers use the same version of Node.js and npm.

Please refer to the following issues for more context: [#327](https://github.com/vitabaks/postgresql_cluster/issues/327) and [#341](https://github.com/vitabaks/postgresql_cluster/issues/341).

Thanks again for your feedback.

Reply:
not once in the years of developing this automation tool have I needed Node.js 

but if you think that it is really necessary (but maybe not for me personally) then let's add it.